# Time stamp
Header header

# Latitude [degrees] - Positive is north
float64 latitude

# Longitude [degrees] - Positive is east
float64 longitude

# Altitude [m]
float32 altitude

# utm data
float32 utm_easting
float32 utm_northing
int16 Zone
string fields